docs/source/references/running.rst

```diff
@@ -0,0 +1,124 @@
+.. _running-the-app:
+
+Successfully Running the DRG
+============================
+
+.. _step-by-step-instr:
+
+Steps to Use the DRG
+--------------------
+
+You can easily standup a DRG in four easy steps!
+
+#. Create a Data Resource Schema -- see :ref:`basic usage<basic-usage>`
+#. Run the Data Resource Generator -- see :ref:`basic usage<basic-usage>` and :ref:`running the application<in-prod-usage>`
+#. Generate your Data Resources -- see :ref:`basic usage<basic-usage>`
+#. Interact with your generated Data Resources -- see :ref:`basic usage<basic-usage>`
+
+Running the application
+-----------------------
+
+Running with Docker is the preferred method. We recommend using docker-compose locally to run the application and using more advanced docker deployment tools such as Docker Swarm or k8s, depending on your needs. Advanced deployment is outside of the scope of this documentation however. Please see `in-production usage <in-prod-usage>`_ for more information.
+
+Running Locally with Docker Compose
+'''''''''''''''''''''''''''''''''''
+
+To run locally via Docker Compose:
+
+#. First build the docker image:
+
+.. code-block:: bash
+
+    docker build -t brighthive/data-resource-generator .
+
+#. Then run the docker-compose.yml file (which will run brighthive/data-resource-generator:latest) with:
+
+.. code-block:: bash
+
+    docker-compose up
+
+Running Locally with Python
+'''''''''''''''''''''''''''
+
+This method is sometimes useful for doing development work.
+
+1. Restart the database to clear the data.
+
+.. code-block:: bash
+
+    docker-compose -f test-database-docker-compose.yml down && docker-compose -f test-database-docker-compose.yml up -d
+
+1. To run the application in production mode,
+
+.. code-block:: bash
+
+    pipenv run python wsgi.py
+
+Or to run in testing mode with flask,
+
+.. code-block:: bash
+
+    pipenv run flask run
+
+.. _starting-the-app:
+
+Starting the application
+------------------------
+
+There are two ways to run the application. With an empty database or an already existing database.
+
+Generation Validation
```

Comment:
Should this be somewhere else? It seems like it does not belong in a section that describes database generation.